src/pages/Landing/index.tsx

```diff
@@ -56,6 +56,29 @@ export default () => {
   const columnHeaders = ['Name', 'Age', 'Is Cool'];
 
   if (isLoading) return <Loader />;
+  function handleDelete(customerId: any): void {
+    const deleteCustomer = () => {
+      const customers = customersGetter();
+      const index = customers.findIndex((customer) => customer.name === customerId);
+
+
+      if (index !== -1) {
+        const updatedCustomers = [...customers];
+        updatedCustomers.splice(index, 1);
+        customersSetter(updatedCustomers);
+      }
+    };
+
+    try {
+      deleteCustomer();
+      queryClient.invalidateQueries('customers');
+    } catch (error) {
+      console.error('An error occurred while deleting the customer:', error);
+
+    }
+  }
```

Comment:
ideally this function would exist in src/api/CustomerApi.ts  and be exported there.  then here you could import it and use it.  Note that it would need to accept name as an argument and you'll pass the name to it.